backend/models/sqlite/sqlite.go

```diff
@@ -380,3 +385,26 @@ func (s *Sqlite) UpdateCluster(cluster *models.Cluster) error {
 func (s *Sqlite) DeleteCluster(userID string, projectName string) error {
 	return s.db.Where("user_id = ? AND project_name = ?", userID, projectName).Delete(&models.Cluster{}).Error
 }
+
+func (s *Sqlite) CreatePendingRecord(record *models.PendingRecord) error {
+	record.CreatedAt = time.Now()
+	return s.db.Create(record).Error
+}
+
+func (s *Sqlite) ListAllPendingRecords() ([]models.PendingRecord, error) {
+	var pendingRecords []models.PendingRecord
+	return pendingRecords, s.db.Find(&pendingRecords).Error
+}
+
+func (s *Sqlite) ListOnlyPendingRecords() ([]models.PendingRecord, error) {
+	var pendingRecords []models.PendingRecord
+	return pendingRecords, s.db.Where("amount > transferred_amount").Find(&pendingRecords).Error
```

Comment:
shouldn't amount be tft_amount according to the PendingRecord struct?

Reply:
https://gorm.io/docs/conventions.html#Column-Name I only saw json tag not a gorm column tag